src/main/resources/SnykBundle.properties

```diff
@@ -0,0 +1,3 @@
+snyk.settings.organization.tooltip.description=An organization groups projects, and can have team members who can access these projects.
+snyk.settings.organization.tooltip.linkText=What’s a Snyk organization
```

Comment:
is question mark missed at the end?

Reply:
asked Andy for confirmation.

Reply:
Andy confirmed that it looks good and as discussed in the JIRA ticket.